tools/run-tests.py

```diff
@@ -196,6 +196,13 @@ def get_arguments():
 
 BINARY_CACHE = {}
 
+TERM_NORMAL = '\033[0m'
+TERM_BLUE = '\033[1;34m'
+
+def report_command(cmd_type, cmd):
+    sys.stderr.write('%s%s%s\n' % (TERM_BLUE, cmd_type, TERM_NORMAL))
+    sys.stderr.write('%s%s%s\n\n' % (TERM_BLUE, ' \\\n\t'.join(cmd), TERM_NORMAL))
```

Comment:
It's weird, but it seems that Travis CI automatically resets the color at the end of every line of the log. This is non-standard and differs from the behaviour of desktop terminals. Still, it makes the logs look strange. The above snippet should be adjusted to make both terminals and Travis logs look nice.